bitcoin/Cargo.toml

```diff
@@ -28,7 +28,7 @@ bitcoinconsensus-std = ["bitcoinconsensus/std", "std"]
 # Instead no-std enables additional features required for this crate to be usable without std.
 # As a result, both can be enabled without conflict.
 std = ["bech32/std", "hashes/std", "hex/std", "internals/std", "io/std", "secp256k1/std"]
-no-std = ["bech32/alloc", "hashes/alloc", "hashes/io", "hex/alloc", "io/alloc", "secp256k1/alloc"]
+no-std = ["bech32/alloc", "hashes/alloc", "hashes/io", "hex/alloc", "internals/alloc", "io/alloc", "secp256k1/alloc"]
```

Comment:
Oh, it does hurt: it increases compilation time. Although arguably it will be needed for better error messages anyway when `bitcoin-units` is added so this is OK.